backend/app/core/service_layer.py

```diff
@@ -28,3 +31,22 @@ def login_user_service(request: WSGIRequest, login_user_vo: LoginUserVO):
         login(request, user)
         return {'user_id': user.id}
     return User.DoesNotExist
+
+
+def serials_list_service(request: WSGIRequest, user_id: SerialsListVO):
+    try:
+        check_auth_service(request)
+        if request.user.id != user_id.user_id:
+            raise PermissionDenied
+    except PermissionDenied:
+        return HttpResponse(status=403)
+    else:
+        serials_query = list(Bookmark.objects.filter(owner=user_id.user_id).values())
+        return json.dumps(serials_query)
```

Comment:
Move "json.dumps" to the controller layer. You dump data into JSON to return it from the endpoint, so this endpoint-related operation, not business-logic-related operation.